### ISSUE TITLE
Improved editing of shopping list

### ISSUE THREAD

**Issue by philip-n** [TandoorRecipes/recipes#1278]
**Is your feature request related to a problem? Please describe.**
If I add some recipes to the shopping list, the ingredients stay linked to the recipes. This is great as it allows scaling the portions for the whole recipe and having the ingredients scale proportionally :+1:

What I miss is a way to edit the ingredients manually. If a recipe requires 7 eggs potatoes and I still have 4 eggs in the fridge, I might want to buy only the 3 missing eggs. Or maybe buy 5, so I have to spare ones. Or buy 8, as I want to cook something else not on the list. Or ...

**Describe the solution you'd like**
What I'd like to be added is two-fold:
1. A way to remove the recipe <-> ingredient relationship, so that each ingredient is unlinked from the recipe and can be manually edited. This would of course freeze the current scaling of the recipe.
2. Be able to change the amount on items. Currently, there is only the "delete" button for manually added items.

**Describe alternatives you've considered**
Multiple: Export the list and edit it elsewhere; add items manually until the desired amount matches; mark some ingredients as "not on shopping list" and add them manually.

However, none of them sounds appealing :wink:

**Comment by smilerz** [comment]
These are good suggestions, and probably not terribly difficult to implement, though may not get to them for a little while.

**Comment by TWART016** [comment]
Change the amount would be great.

**Comment by chernesk** [comment]
Would love this as well. 

My use case is coming from Paprika, it was awesome to be able to have a list come up of the ingredients that are in the recipe and then I can uncheck the ones that I don't want added to the shopping list because I know I already have them.

Currently, I just check them off before I go shopping. Not ideal, but it works.

**Comment by vabene1111** [comment]
@chernesk i think the new shopping list can do exactly that. currently this does not yet work when automatically adding from the meal plan but thats something on the todo list as well.

**Comment by chernesk** [comment]
I'll check that out. I generally only add from the meal plan.

**Comment by TWART016** [comment]
Are there new information about that requirement?

**Comment by vabene1111** [comment]
no, we generally work on issues when we have the time to do so and there are currently things we find more important than this thus they will be implemented first
